test/source/mock/backend/backend-data.ts

```diff
@@ -117,6 +119,19 @@ export class BackendData {
         allow_attester_search_only_for_domains: [],
       };
     }
+    if (domain === 'only-allow-some-domains-for-keys-openpgp-org-client-configuration.flowcrypt.test') {
+      return {
+        flags: [],
+        allow_keys_openpgp_org_search_only_for_domains: ['flowcrypt.com'],
```

Comment:
A test domain would be better, like `allowed-domain.test`

Reply:
Got it